src/dialects/oracledb/schema/columncompiler.js

```diff
@@ -12,12 +12,24 @@ inherits(ColumnCompiler_Oracledb, ColumnCompiler_Oracle);
 assign(ColumnCompiler_Oracledb.prototype, {
   time: 'timestamp with local time zone',
 
-  datetime: function(without) {
-    return without ? 'timestamp' : 'timestamp with local time zone';
+  datetime: function(withoutTz) {
+    let useTz;
+    if (isObject(withoutTz)) {
+      ({ useTz } = withoutTz);
+    } else {
+      useTz = !withoutTz;
+    }
+    return useTz ? 'timestamp with local time zone' : 'timestamp';
   },
 
-  timestamp: function(without) {
-    return without ? 'timestamp' : 'timestamp with local time zone';
+  timestamp: function(withoutTz) {
```

Comment:
This will need to wait until oracle tests are back online + testcase

Reply:
Given that the outcome is identical I don't see why it will need to wait, but its up to you.